src/main/java/uk/gov/companieshouse/reconciliation/function/compare_count/CompareCountRoute.java

```diff
@@ -45,14 +53,30 @@ public void configure() {
                     exchange.getIn().setHeader("Weight", weight);
                     exchange.getIn().setHeader("WeightAbs", weight.abs());
                 })
+                .enrich().simple(("${header.Src}")).aggregationStrategy((base, src) -> {
+                    base.getIn().setHeader("SrcList", new ResourceList(Collections.singletonList(src.getIn().getHeader("SrcCount", String.class)),
+                            src.getIn().getHeader("SrcName", String.class)));
+                    return base;
+                })
+                .enrich().simple("${header.Target}").aggregationStrategy((base, target) -> {
+                    base.getIn().setHeader("TargetList", new ResourceList(Collections.singletonList(target.getIn().getHeader("TargetCount", String.class)),
+                            target.getIn().getHeader("TargetName", String.class)));
+                    return base;
+                })
+                .bean(CompareCountTransformer.class)
+                .marshal().csv()
+                .toD("${header.Upload}")
+                .toD("${header.Presign}")
+                .setHeader("CompareCountLink", body())
```

Comment:
Can be simplified by using a single header to store all values and descriptions across all pipelines.